cli/using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,104 @@
+
+.Procedure
+
+* Run the following command to get information about your current authentication status and user credentials in the Central system:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl central whoami
+----
++
+.Example output
++
+[source,terminal,subs="+quotes"]
+----
+UserID:
+	    <redacted>
+User name:
+	    <redacted>
+Roles:
+ APIToken creator, Admin, Analyst, Continuous Integration, Network Graph Viewer, None, Sensor Creator, Vulnerability Management Approver, Vulnerability Management Requester, Vulnerability Manager, Vulnerability Report Creator
+Access:
+  rw Access
+  rw Administration
+  rw Alert
+  rw CVE
+  rw Cluster
+  rw Compliance
+  rw Deployment
+  rw DeploymentExtension
+  rw Detection
+  rw Image
+  rw Integration
+  rw K8sRole
+  rw K8sRoleBinding
+  rw K8sSubject
+  rw Namespace
+  rw NetworkGraph
+  rw NetworkPolicy
+  rw Node
+  rw Secret
+  rw ServiceAccount
+  rw VulnerabilityManagementApprovals
+  rw VulnerabilityManagementRequests
+  rw WatchedImage
+  rw WorkflowAdministration
+----
++
+Review the output to ensure that the authentication and user details are as expected.
+
+[id="authenticating-by-using-the-roxctl-cli_{context}"]
+== Authenticating by using the roxctl CLI
+
+For authentication, you can use an API token, your administrator password, or the `roxctl login` command.
+
+The following are some guidelines for the effective use of API tokens:
+
+* Use an API token in a production environment with continuous integration (CI). Each token is assigned specific access permissions, providing control over the actions it can perform. In addition, API tokens do not require interactive processes, such as browser-based logins, making them ideal for automated processes. These tokens have a time-to-live (TTL) of 1 year, providing a longer validity period for seamless integration and operational efficiency.
+
+* Use your administrator password only for testing purposes. Do not use it in the production environment.
+
+* Use the `roxctl login` command only for interactive, local uses.
+
+//Creating an API token
+include::modules/create-api-token.adoc[leveloffset=+2]
+
+//Exporting and saving the API token
+include::modules/cli-authentication.adoc[leveloffset=+2]
+
+//Configuring an authentication provider in Central
+include::modules/configuring-an-authentication-provider-in-central.adoc[leveloffset=+2]
+
+
+[id="configuring-and-using-the-roxctl-cli-in-rhacs-cloud-service_{context}"]
+== Configuring and using the roxctl CLI in {product-title-managed-short}
+
+.Procedure
+
+* Export the following variables before using these commands:
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_API_TOKEN=_<api_token>_
+----
++
+[source,terminal,subs="+quotes"]
+----
+$ export ROX_CENTRAL_ADDRESS=_<address>:<port_number>_
```

Comment:
Let's use `ROX_ENDPOINT` instead. This environment variable is the replacement of the `-e` flag and its recommended to use the environment variable instead of the flag.

The environment variable `ROX_CENTRAL_ADDRESS` is something just created within this doc, it is neither understood nor supported by roxctl and has caused confusion in the past (people set `ROX_CENTRAL_ADDRESS` and expected `roxctl` to retrieve the address from the environment variable). The correct way is to use `ROX_ENDPOINT` and we should use this here.